src/client/core/utils/position.js

```diff
@@ -305,6 +305,15 @@ export function getElementClientRectangle (el) {
     };
 }
 
+export function getElementsRelation (dimensions1, dimensions2) {
```

Comment:
Position of the element related to the other one. For example, `relation.left = -200` means that the left border of the `element1` is to the 200px left from the left border of the `element2`. Maybe  you have more appropriate naming for this?